doc/src/modules/solvers/solveset.rst

```diff
@@ -315,19 +317,19 @@ What are the general methods employed by solveset to solve an equation?
    used as follows:
 
    - The method to solve the rational function is called:
-     ``_solve_as_rational(f, symbol, solveset_solver, as_poly_solver)``
-     its third argument is the ``solveset_solver`` which can either be
-     ``solveset_real`` or ``solveset_complex`` based on these, respective
-     poly solvers ``_solve_as_poly_real`` or ``_solve_as_poly_complex``
-     is called to solve as polynomial.
+     ``_solve_as_rational(f, symbol, solveset_solver, as_poly_solver)`` its
+     third argument is the ``solveset_solver`` which can either be
+     ``solveset_real`` or ``solveset_complex`` based on these, the respective
+     poly solver ``_solve_as_poly_real`` or ``_solve_as_poly_complex`` is
+     called to solve as polynomial.
 
    - The underlying method ``_solve_as_poly`` solves the equation using
-     polynomial techniques if its already a polynomial equation or, with
+     polynomial techniques if it's already a polynomial equation or, with
      a change of variables, can be made so.
 
 
- * The final solution set obtained is taken intersection with the input
-   domain, and the resulting solution is returned.
+ * The final solution set is intersected with the input domain, and the
```

Comment:
I think I prefer the wording here. It makes it clear that the intersecting is an action taken as the final step.